gradle/dependency-versions.gradle

```diff
@@ -19,7 +19,7 @@ ext {
     javaxVersion = "3.0.1"
     nettyVersion = "4.1.74.Final"
     nettyTcnativeVersion = "2.0.49.Final"
-    helixVersion = "1.0.1"
```

Comment:
Please don't use 1.0.3. This version will be deprecated.

Reply:
Ack. Will wait for the next Helix release.